decidim-core/db/migrate/20170123140857_add_avatar_to_user_groups.rb

```diff
@@ -0,0 +1,5 @@
+class AddAvatarToUserGroups < ActiveRecord::Migration[5.0]
+  def change
+    add_column :decidim_user_groups, :avatar, :string
```

Comment:
We don't have support for avatars right now. I'd just return a placeholder image (like we're doing with users) and add the whole uploading a logo feature later.